sql/mob_pools.sql

```diff
@@ -2147,7 +2147,7 @@ INSERT INTO `mob_pools` VALUES (2086,'Invulnerable_Mazzgozz','Invulnerable_Mazzg
 INSERT INTO `mob_pools` VALUES (2087,'Iqi-Balam','Iqi-Balam',71,0x0000C60800000000000000000000000000000000,4,4,12,240,100,0,1,0,0,2,0,0,0,3,0,0,2,1,0,71);
 INSERT INTO `mob_pools` VALUES (2088,'Irate_Sheep','Irate_Sheep',226,0x0000540100000000000000000000000000000000,1,1,11,240,100,0,0,0,1,0,0,0,0,0,0,0,0,1,0,226);
 INSERT INTO `mob_pools` VALUES (2089,'Iriri_Samariri','Iriri_Samariri',297,0x0000150700000000000000000000000000000000,4,4,5,150,100,0,0,0,0,2,0,0,0,3,0,0,2,1,0,297);
-INSERT INTO `mob_pools` VALUES (2090,'Iriz_Ima','Iriz_Ima',295,0x0000CA0600000000000000000000000000000000,1,1,7,320,100,0,1,0,0,2,256,32,7976,1181,0,0,0,1,0,295);
+INSERT INTO `mob_pools` VALUES (2090,'Iriz_Ima','Iriz_Ima',295,0x0000CA0600000000000000000000000000000000,1,1,7,320,100,0,1,0,0,2,256,32,7976,1181,4,0,0,0,0,295);
```

Comment:
isn't that third-to-last number the "name visible" flag?  Is Iriz Ima's name invisible?

Reply:
Yes. But im not sure what that column actually makes the name invis. Its from the mob capture. I can test again in a few cant remember if the name was still visable lol

Reply:
[16986429] = {['id']=16986429, ['name']="IrizIma", ['polutils_name']="Iriz Ima", ['npc_type']="Simple NPC", ['index']=317, ['x']=257.463, ['y']=-23.929, ['z']=120.741, ['r']=120, ['flag']=72, ['speed']=24, ['speedsub']=70, ['animation']=1, ['animationsub']=4, ['namevis']=0, ['status']=1, ['flags']=1181, ['name_prefix']=32, ['look']="0000CA06", ['raw_packet']="0E2034023D3103013D010F7844BB8043986EBFC1647BF1424800000018466401019D0400001B0020000004008AC307000000CA064972697A496D610000000000"},

Reply:
![image](https://user-images.githubusercontent.com/35616771/53023323-9bdd1300-3422-11e9-8716-6dd3e7d9481a.png)
name still visable

Reply:
Just making sure: You reran the SQL when checking namevis?

I see NPCLogger claims namevis is 0, and I guess I should be the last person to doubt my own addon. But still.

If you reran SQL... I dunno. Maybe the part of the packet we call "namevis" isn't always just a simple true/false value.

Reply:
Correct teo says the name is ran on the entityflags instead

Reply:
> [9:47 AM] Teo: namvis is kinda retardedly in both its own column and a flag bit in dsp..and not sure which is

Reply:
https://github.com/DarkstarProject/darkstar/search?q=namevis&unscoped_q=namevis

Reply:
tl;dr "we don't know wtf we are even doing and probably broke it anyway"